zeebe/exporters/opensearch-exporter/src/main/java/io/camunda/zeebe/exporter/opensearch/OpensearchClient.java

```diff
@@ -235,6 +236,19 @@ public boolean updateIndexStateManagementPolicy(final Integer seqNo, final Integ
     return putIndexStateManagementPolicy(queryParameters);
   }
 
+  public boolean deleteIndexStateManagementPolicy() {
+    try {
+      final var request =
+          new Request(
+              "DELETE", "_plugins/_ism/policies/" + configuration.retention.getPolicyName());
+
+      final var response = sendRequest(request, DeleteStateManagementPolicyResponse.class);
+      return response.result().equals(DeleteStateManagementPolicyResponse.DELETED);
```

Comment:
❓ so in case it doesn't exist this does not end up in a 404 but just returns a result. The [API docs](https://opensearch.org/docs/latest/im-plugin/ism/api/#delete-policy) don't really answer that :/